data.json

```diff
@@ -520,5 +520,11 @@
     "githubUsername": "webdevrachit",
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
+  },
+  {
+    "name": "Ivan Pesenti",
+    "githubUsername": "ivan-pesenti",
+    "imageUrl": "https://user-images.githubusercontent.com/60645922/106567973-ea621a00-6532-11eb-949b-4999700efce1.png",
+    "issueNumber": 308
   }
-]
+]
```

Comment:
I think we need a new line here :nerd_face:

Reply:
I don't understand why it has failed this check.... I think there was no syntax errors in the file.

Reply:
I ran prettier on your fork. It's flagging a warning because there is no new line at the end of the file.
```
diff --git a/data.json b/data.json
index fe9ba6d..d9674be 100644
--- a/data.json
+++ b/data.json
@@ -527,4 +527,4 @@
     "imageUrl": "https://user-images.githubusercontent.com/60645922/106567973-ea621a00-6532-11eb-949b-4999700efce1.png",
     "issueNumber": 308
   }
-]
\ No newline at end of file
+]
```

You can test this yourself by running `npx prettier -w data.json` and `git diff`

If you need any help let me know :nerd_face: